spec/client_spec.rb

```diff
@@ -92,6 +92,22 @@
         expect(lock_manager).to receive(:sleep).exactly(lock_manager_opts[:retry_count] - 1).times
         lock_manager.lock(resource_key, ttl)
       end
+
+      it 'sleeps at least the specified retry_delay in milliseconds' do
```

Comment:
I don't feel comfortable stubbing so much, being in the range of expected values shouldn't be broken. Does it make sense? I have no problem on setting a explicit value but in that case this spec does no make sense, but we can do one that just assert we receive the sum of both values